### ISSUE TITLE
[CV] Request new entry: base peak intensity maximum until RT

### ISSUE THREAD

**Issue by julianu** [HUPO-PSI/mzQC#247]
### What is the QC term's name?

base peak intensity maximum until RT

### Briefly describe the QC term.

this term is similar to #246 but only takes the BPs into account until a given retention time.
This could maybe be modeled using a tuple or table specifying the RT and the maximum BP until this RT
Also #246 could be modelled using e.g. "-1" as RT to specify "global maximum BP"

### What is the QC term's unit?

intensity / arbitrary and time (in seconds)

### Value type

MS:4000004 ! n-tuple

### Describe any additional information.

This metric is similar to #246 , but can cut away a very high gradient change in the end of an MS run.

**Comment by julianu** [comment]
Generalizing for a RT interval makes perfectly sense.
The idea behind this was, that we look at the BP in a certain RT range, which does not include the very strong elution step / increase of gradient common at the end of a run's RT.

**Comment by julianu** [comment]
I generalized it a bit more, so that the table may contain several rows.

Recommended term:

```
[Term]
id: MS:4000xyz
name: maximum base peak intensity in RT range
def: "This metric reflects the maximum base peak intensity (MS:1000505) of all spectra in a single run in given retention time windows. The retention time windows are specified by the 2nd column defining the start time in seconds and the 3rd column defining the end time in seconds. To only define a starting time, use -1 in the 3rd column." [PSI:MS]
is_a: MS:4000005 ! table
relationship: has_column MS:1000505 ! base peak intensity
relationship: has_column UO:0000010 ! second
relationship: has_column UO:0000010 ! second
relationship: has_metric_category MS:4000009 ! ID free metric
relationship: has_metric_category MS:4000012 ! single run based metric
relationship: has_metric_category MS:4000015 ! multiple spectra based metric
```

**Comment by bittremieux** [comment]
- I'd maybe write "retention time" in full in the term's name.
- Discussion point: Should the time-based columns maybe be `MS:1000016 ! scan start time` instead?

Otherwise it looks good to me.

**Comment by julianu** [comment]
I thought of using the `MS:1000016 ! scan start time` as well, but it's weird to use this as the encoding of the end time. Hence I would prefer to use just the `UO:0000010 ! second`, which will also grant the exact unit.

Will be added to #263

**Comment by bittremieux** [comment]
Maybe we should introduce two new terms to record a time range (or a single term that takes two values in a tuple)? The duplicate columns seem a bit unsatisfying, because there's no automatic difference between both columns based on the term definition. So if their order gets mixed up, the values don't make sense anymore, or might even be just wrong.

**Comment by julianu** [comment]
I agree. I will check the ontology whether there is something like a "range start", "range end" and a "duration". If not, I will request a new term.
